storeProjectHW/storeProjectHW/PersonViewController.swift

```diff
@@ -8,9 +8,413 @@
 import UIKit
 /// Экран пользователя
 final class PersonViewController: UIViewController {
+    private enum Constants {
+        static let forYouText = "Для вас"
+        static let newsText = "Вот что нового"
+        static let devicesText = "ваши устройства"
+        static let avatarKey = "avatar"
+        static let showAllText = "Показать все"
+        static let recomendText = "Рекомендуется вам"
+        static let recomentNewsText = "Получайте новости в своем заказе в режиме реального времени."
+        static let recomendNotificationText = "Включите уведомления, чтобы получить новости о своем заказе."
+        static let treatmentText = "Обрабатывается"
+        static let sendedText = "Отправлено"
+        static let deliveredText = "Доставлено"
+        static let orderText = "Ваш заказ отправлен."
+        static let countOrderText = "1 товар, доставка завтра"
+        static let mainTextColor = "mainTextColor"
+        static let searchColor = "searchaColor"
+        static let tabColor = "tabColor"
+        static let viewBackColor = "viewBackColor"
+        static let chevronForward = "chevron.forward"
+        static let airpodsImageName = "apple-airpod"
+        static let recommendedImageName = "app.badge"
+        static let avatarImageName = "person.fill"
+ 
+        static let ImageSizeForLargeState: CGFloat = 40
+        static let ImageRightMargin: CGFloat = 16
+        static let ImageBottomMarginForLargeState: CGFloat = 12
+        static let ImageBottomMarginForSmallState: CGFloat = 6
+        static let ImageSizeForSmallState: CGFloat = 32
+        static let NavBarHeightSmallState: CGFloat = 44
+        static let NavBarHeightLargeState: CGFloat = 96.5
+    }
+    
+    // MARK: - Private Visual Components
+    private lazy var mainScrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.frame = view.frame
+        scrollView.contentInset = UIEdgeInsets(top: -200, left: 0, bottom: 0, right: 0)
+        scrollView.contentSize = CGSize(width: view.bounds.width, height: 1000)
+        
+        return scrollView
+    }()
+     
+    private lazy var mainImageView: UIImageView = {
```

Comment:
```suggestion
    private let mainImageView: UIImageView = {
```
lazy var оставляем только там где требует компилятор